test/functionalTest/cases/117_convop_using_standard_ops/getAttributeValueInstance.test

```diff
@@ -0,0 +1,335 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+ConvOp getAttributeValueInstance: GET /v1/contextEntities/{entityId::id}/attributes/{attribute::name}/{matadata::Id}
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0
+
+--SHELL--
+
+#
+# 01. GET /v1/contextEntities/E1/attributes/A1/ID_A?entity::type=T1 and see it fail
+# 02. Create E1/T1/A1 with metadata ID ID_A
+# 03. GET /v1/contextEntities/E1/attributes/A1/ID_A?entity::type=T1 and see it work
+# 04. GET /v1/contextEntities/E1/attributes/A1/ID_B?entity::type=T1 and see it fail
+# 05. GET /v1/contextEntities/E1/attributes/A1/ID_A?entity::type=T1&attributesFormat=object
+# 06. Create E1//A1 with metadata ID ID_B
+# 07. GET /v1/contextEntities/E1/attributes/A1/ID_B?!exist=entity::type and see it work
```

Comment:
Looking to the list of supported arguments in getAttributeValueInstance() preamble, it seems that `?exist=entity::type` is not covered by these steps.